test/utils/tablify.js

```diff
@@ -0,0 +1,251 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2017–2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import tablify from '../../src/utils/tablify';
+import { objectToKeyValueString } from '../utils';
+
+describe('tablify utils', () => {
+	describe('when an empty object is used', () => {
+		it('should have 0 length', () => {
+			const printValue = {};
+			const returnValue = tablify(printValue);
+			return expect(returnValue).to.have.length(0);
+		});
+	});
+
+	describe('a non-empty object', () => {
+		it('should have the matching rows', () => {
+			const printValue = {
+				lisk: 'js',
+				version: 1,
+			};
+			const returnValue = tablify(printValue);
+			return Object.entries(printValue).forEach(([key, value], arrayKey) => {
+				expect({ [key]: value }).to.eql(returnValue[arrayKey]);
+			});
+		});
+	});
+
+	describe('a nested object', () => {
+		it('should have the matching rows', () => {
+			const printValue = {
+				root: 'value',
+				nested: {
+					object: 'values',
+					testing: 123,
+					nullValue: null,
+					keys: {
+						more: ['publicKey1', 'publicKey2'],
+					},
+				},
+			};
+			const returnValue = tablify(printValue);
+			return Object.entries(printValue).forEach(([key, value], arrayKey) => {
+				const strValue =
+					typeof value === 'object' && value !== null
+						? Object.entries(value)
```

Comment:
Can we extract this into a function outside the ternary for clarity?

Reply:
This is the one we have utils for =0